README.md

```diff
@@ -15,7 +15,7 @@ npm install --save swagger-test-templates
 Use your [Swagger](http://swagger.io) API spec file to generate test for your API.
 
 ```javascript
-var testGen = require('swagger-test-templates');
```

Comment:
Yep, considered that too... Just thought 'stt' will be useful if we export more functions in future...

Your call :-)

Cheers,
Anil Sagar